stock_picking_update_responsible/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Stock Picking Update Responsible",
```

Comment:
According to new requirements, we should split this addon in two (one for pure stock stuff, and another one with sales additions), or directly rename this, as unique addon. We'll start with this second option:
```suggestion
    "name": "Sale Stock Picking Update Responsible",
```